TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TeleNew.java

```diff
@@ -38,134 +38,87 @@ public void init() {
-        }
-        if (gamepad1.left_stick_x < 0.3) {
-            wheel1.setPower(0.7);
-            wheel2.setPower(0.7);
-            wheel3.setPower(-0.7);
-            wheel4.setPower(-0.7);
-        }
-        if (gamepad1.left_stick_x > -0.3) {
-            wheel1.setPower(-0.7);
-            wheel2.setPower(-0.7);
-            wheel3.setPower(0.7);
-            wheel4.setPower(0.7);
-        }
+            //Neutral condition for horizontal
+        if (gamepad1.left_stick_x > -0.3 && gamepad1.left_stick_x < 0.3) { wheelMove (0, 0, 0, 0); }
 
-        if (gamepad1.left_bumper) {
-            wheel1.setPower(0.6);
-            wheel2.setPower(0.6);
-            wheel3.setPower(0.6);
-            wheel4.setPower(0.6);
-        }
-        if (gamepad1.right_bumper) {
-            wheel1.setPower(-0.6);
-            wheel2.setPower(-0.6);
-            wheel3.setPower(-0.6);
-            wheel4.setPower(-0.6);
-        }
+            //Go up
+        if (gamepad1.left_stick_y < 0.3) { wheelMove (0.7, -0.7, 0.7, -0.7); }
 
-       if (gamepad1.right_stick_y > -0.3 && gamepad1.right_stick_y < 0.3) {
-            wheel1.setPower(0);
-            wheel2.setPower(0);
-            wheel3.setPower(0);
-            wheel4.setPower(0);
-        }
-        if (gamepad1.right_stick_x > -0.3 && gamepad1.left_stick_x < 0.3) {
-            wheel1.setPower(0);
-            wheel2.setPower(0);
-            wheel3.setPower(0);
-            wheel4.setPower(0);
-        }
-        if (gamepad1.right_stick_y < 0.3) {
-            wheel1.setPower(0.35);
-            wheel2.setPower(-0.35);
-            wheel3.setPower(0.35);
-            wheel4.setPower(-0.35);
-        }
-        if (gamepad1.right_stick_y > -0.3) {
-            wheel1.setPower(-0.35);
-            wheel2.setPower(0.35);
-            wheel3.setPower(-0.35);
-            wheel4.setPower(0.35);
-        }
-        if (gamepad1.right_stick_x < 0.3) {
-            wheel1.setPower(0.35);
-            wheel2.setPower(0.35);
-            wheel3.setPower(-0.35);
-            wheel4.setPower(-0.35);
-        }
-        if (gamepad1.right_stick_x > -0.3) {
-            wheel1.setPower(-0.35);
-            wheel2.setPower(-0.35);
-            wheel3.setPower(0.35);
-            wheel4.setPower(0.35);
-        }
+            //Go down
+        if (gamepad1.left_stick_y > -0.3) { wheelMove (-0.7, 0.7, -0.7, 0.7); }
 
-        if (gamepad1.left_trigger == 1) {
-            claw.setPosition(openClaw);
-        }
+            //Go right
+        if (gamepad1.left_stick_x < 0.3) { wheelMove (0.7, 0.7, -0.7, -0.7); }
 
-        if (gamepad1.right_trigger == 1) {
-            claw.setPosition(closeClaw);
-        }
+            //Go left
+        if (gamepad1.left_stick_x > -0.3) { wheelMove (-0.7, -0.7, 0.7, 0.7); }
 
-        if (gamepad2.left_bumper) {
-            claw.setPosition(openClaw);
-        }
+            //Rotate Left
+        if (gamepad1.left_bumper) { wheelMove (0.6, 0.6, 0.6, 0.6); }
```

Comment:
What happens if this is pressed at the same time as the stick?

Reply:
Both should execute at once from my knowledge of the loop function used here it is nearly instantaneous and looks like the two are going at the same time but its going one after another.